modules/configuration/pages/about.adoc

```diff
@@ -239,14 +239,14 @@ If, for example, we wanted to generate a config with a websocket input, a Kafka
 
 [,bash,subs="attributes+"]
 ----
-{rpk-command} create websocket/mapping/kafka
+rpk connect create websocket/mapping/kafka
 ----
 
-TIP: To see which components {page-component-title} offers, use `{rpk-command} list`.
+TIP: To see which components {page-component-title} offers, use `rpk connect run list`.
```

Comment:
Looks like this should have `run` removed as well.
```suggestion
TIP: To see which components {page-component-title} offers, use `rpk connect list`.
```


`$ rpk connect run list`
`Configuration file read error: list: open list: no such file or directory`


<details>
<summary>`rpk connect list`</summary>
<br>

```
Inputs:
  - amqp_0_9
  - amqp_1
  - aws_kinesis
  - aws_s3
  - aws_sqs
  - azure_blob_storage
  - azure_cosmosdb
  - azure_queue_storage
  - azure_table_storage
  - batched
  - beanstalkd
  - broker
  - cassandra
  - cockroachdb_changefeed
  - csv
  - discord
  - dynamic
  - file
  - gcp_bigquery_select
  - gcp_cloud_storage
  - gcp_pubsub
  - generate
  - hdfs
  - http_client
  - http_server
  - inproc
  - kafka
  - kafka_franz
  - mongodb
  - mqtt
  - nanomsg
  - nats
  - nats_jetstream
  - nats_kv
  - nats_stream
  - nsq
  - parquet
  - pulsar
  - read_until
  - redis_list
  - redis_pubsub
  - redis_scan
  - redis_streams
  - resource
  - sequence
  - sftp
  - socket
  - socket_server
  - sql_raw
  - sql_select
  - stdin
  - subprocess
  - twitter_search
  - websocket

Processors:
  - archive
  - avro
  - awk
  - aws_dynamodb_partiql
  - aws_lambda
  - azure_cosmosdb
  - bloblang
  - bounds_check
  - branch
  - cache
  - cached
  - catch
  - command
  - compress
  - couchbase
  - decompress
  - dedupe
  - for_each
  - gcp_bigquery_select
  - grok
  - group_by
  - group_by_value
  - http
  - insert_part
  - javascript
  - jmespath
  - jq
  - json_schema
  - log
  - mapping
  - metric
  - mongodb
  - msgpack
  - mutation
  - nats_kv
  - nats_request_reply
  - noop
  - parallel
  - parquet_decode
  - parquet_encode
  - parse_log
  - processors
  - protobuf
  - rate_limit
  - redis
  - redis_script
  - resource
  - retry
  - schema_registry_decode
  - schema_registry_encode
  - select_parts
  - sentry_capture
  - sleep
  - split
  - sql_insert
  - sql_raw
  - sql_select
  - subprocess
  - switch
  - sync_response
  - try
  - unarchive
  - wasm
  - while
  - workflow
  - xml

Outputs:
  - amqp_0_9
  - amqp_1
  - aws_dynamodb
  - aws_kinesis
  - aws_kinesis_firehose
  - aws_s3
  - aws_sns
  - aws_sqs
  - azure_blob_storage
  - azure_cosmosdb
  - azure_queue_storage
  - azure_table_storage
  - beanstalkd
  - broker
  - cache
  - cassandra
  - discord
  - drop
  - drop_on
  - dynamic
  - elasticsearch
  - fallback
  - file
  - gcp_bigquery
  - gcp_cloud_storage
  - gcp_pubsub
  - hdfs
  - http_client
  - http_server
  - inproc
  - kafka
  - kafka_franz
  - mongodb
  - mqtt
  - nanomsg
  - nats
  - nats_jetstream
  - nats_kv
  - nats_stream
  - nsq
  - opensearch
  - pulsar
  - pusher
  - redis_hash
  - redis_list
  - redis_pubsub
  - redis_streams
  - reject
  - reject_errored
  - resource
  - retry
  - sftp
  - snowflake_put
  - socket
  - splunk_hec
  - sql_insert
  - sql_raw
  - stdout
  - subprocess
  - switch
  - sync_response
  - websocket

Caches:
  - aws_dynamodb
  - aws_s3
  - couchbase
  - file
  - gcp_cloud_storage
  - lru
  - memcached
  - memory
  - mongodb
  - multilevel
  - nats_kv
  - noop
  - redis
  - ristretto
  - sql
  - ttlru

Rate Limits:
  - local
  - redis

Buffers:
  - memory
  - none
  - sqlite
  - system_window

Metrics:
  - aws_cloudwatch
  - influxdb
  - json_api
  - logger
  - none
  - prometheus
  - statsd

Tracers:
  - gcp_cloudtrace
  - jaeger
  - none
  - open_telemetry_collector

Scanners:
  - avro
  - chunker
  - csv
  - decompress
  - json_documents
  - lines
  - re_match
  - skip_bom
  - switch
  - tar
  - to_the_end

Bloblang Functions:
  - batch_index
  - batch_size
  - content
  - counter
  - deleted
  - env
  - error
  - errored
  - fake
  - file
  - file_rel
  - hostname
  - json
  - ksuid
  - metadata
  - nanoid
  - nothing
  - now
  - random_int
  - range
  - snowflake_id
  - throw
  - timestamp_unix
  - timestamp_unix_micro
  - timestamp_unix_milli
  - timestamp_unix_nano
  - tracing_id
  - tracing_span
  - ulid
  - uuid_v4
  - var

Bloblang Methods:
  - abs
  - all
  - any
  - append
  - apply
  - assign
  - bloblang
  - bool
  - bytes
  - capitalize
  - catch
  - ceil
  - collapse
  - compare_argon2
  - compare_bcrypt
  - compress
  - concat
  - contains
  - decode
  - decompress
  - decrypt_aes
  - diff
  - encode
  - encrypt_aes
  - enumerated
  - escape_html
  - escape_url_query
  - exists
  - explode
  - filepath_join
  - filepath_split
  - filter
  - find
  - find_all
  - find_all_by
  - find_by
  - flatten
  - float32
  - float64
  - floor
  - fold
  - format
  - format_json
  - format_msgpack
  - format_xml
  - format_yaml
  - from
  - from_all
  - geoip_anonymous_ip
  - geoip_asn
  - geoip_city
  - geoip_connection_type
  - geoip_country
  - geoip_domain
  - geoip_enterprise
  - geoip_isp
  - get
  - has_prefix
  - has_suffix
  - hash
  - index
  - index_of
  - int16
  - int32
  - int64
  - int8
  - join
  - json_path
  - json_schema
  - key_values
  - keys
  - length
  - log
  - log10
  - lowercase
  - map
  - map_each
  - map_each_key
  - max
  - merge
  - min
  - not
  - not_empty
  - not_null
  - number
  - or
  - parse_csv
  - parse_duration
  - parse_duration_iso8601
  - parse_form_url_encoded
  - parse_json
  - parse_jwt_es256
  - parse_jwt_es384
  - parse_jwt_es512
  - parse_jwt_hs256
  - parse_jwt_hs384
  - parse_jwt_hs512
  - parse_jwt_rs256
  - parse_jwt_rs384
  - parse_jwt_rs512
  - parse_msgpack
  - parse_parquet
  - parse_url
  - parse_xml
  - parse_yaml
  - patch
  - quote
  - re_find_all
  - re_find_all_object
  - re_find_all_submatch
  - re_find_object
  - re_match
  - re_replace
  - re_replace_all
  - replace
  - replace_all
  - replace_all_many
  - replace_many
  - reverse
  - round
  - sign_jwt_es256
  - sign_jwt_es384
  - sign_jwt_es512
  - sign_jwt_hs256
  - sign_jwt_hs384
  - sign_jwt_hs512
  - sign_jwt_rs256
  - sign_jwt_rs384
  - sign_jwt_rs512
  - slice
  - slug
  - sort
  - sort_by
  - split
  - squash
  - string
  - strip_html
  - sum
  - trim
  - trim_prefix
  - trim_suffix
  - ts_add_iso8601
  - ts_format
  - ts_parse
  - ts_round
  - ts_strftime
  - ts_strptime
  - ts_sub
  - ts_sub_iso8601
  - ts_tz
  - ts_unix
  - ts_unix_micro
  - ts_unix_milli
  - ts_unix_nano
  - type
  - uint16
  - uint32
  - uint64
  - uint8
  - unescape_html
  - unescape_url_query
  - unique
  - unquote
  - uppercase
  - values
  - with
  - without
  - zip
```
</details>